src/components/Table/hoc/withTableSorting/SortIndicator/SortIndicator.scss

```diff
@@ -3,6 +3,8 @@
 $block: '.#{variables.$ns}sort-indicator';
 
 #{$block} {
+    display: flex;
+
     &__icon {
         vertical-align: -2px;
```

Comment:
It seems that for now this style is redundant.

Reply:
127920ef77c007cbe38e71805229c977c98fb006